client/sites-dashboard/components/sites-dashboard.tsx

```diff
@@ -199,6 +199,9 @@ export function SitesDashboard( {
 							<JetpackLogo className="gridicon" size={ 18 } />
 							<span>{ __( 'Add Jetpack to a self-hosted site' ) }</span>
 						</PopoverMenuItem>
+						<PopoverMenuItem href="/start?source=sites-dashboard&ref=smp-import" icon="arrow-down">
```

Comment:
@wojtekn, what do you think if we use the `TRACK_SOURCE_NAME` variable?

Reply:
@sejas @vykes-mac thanks, I've changed how we generate all three `href` attributes and used the `addQueryArgs` function. I've re-used the existing constant as well.